dev-docs/pb10-notes.md

```diff
@@ -0,0 +1,111 @@
+| zetaBidAdapter | zeta_globalBidAdapter |
+| fanAdapter | fanBidAdapter |
+| cadentaperturemxBidAdapter | cadent_aperture_mxBidAdapter |
+| epomDspBidAdapter | epom_dspBidAdapter |
+| pubwiseBidAdapter | pwbidBidAdapter |
+
+## Consent and Data Handling
+
+* **Default behavior for publisher purpose permissions in the TCF control module now enables purposes P4, P7 and special feature 1.**
+* Global vendor list IDs have been filled in for a number of bidder and analytics modules.
+* **A new activity control, and purpose 1 enforcement, prevent bidder endpoint access to third party storage via set-cookie headers.**
+* **The storage disclosures module enables publishers to identify all keys used in the first party and deny access to undisclosed keys. A build artifact is produced to help provide clear and concise information on device storage use for e-privacy directive adherence.**
+
+## User Id Module
+
+* **The user ID module introduces an `enforceStorageType` flag, which why by default warn when a userId submodule accesses the incorrect storage type. Future versions will prevent access.**
+* **`userId` accepts two new config flags, autoRefresh (default false) and retainConfig (default true).** With `autoRefresh: true`, `userId` automatically refreshes IDs for which the configuration changed (either a previously configured module now has different config, or a new module was configured). With `retainConfig: false`, `userId` "forgets" userIds that were previously configured, but are missing from `userSync.userIds[]` in a later setConfig.
+* For bidders: `bid.userId` is no longer populated; bid modules should rely on `userIdAsEids` or `user.ext.eids`. Several bid adapters made this change in the 10.0 release; others in 9.x in anticipation.
+* Eids in `user.eids` are appended to the array in `user.ext.eids` and de-duplicated.
+
+## TypeScript and Build Updates
+
+* **TypeScript files are now accepted in the code base. Tooling and linting were updated accordingly.**
+* The build target browser use statistics are now updated as part of the build process.
+* Tests now target Chrome 109 as the minimum version. `not dead` was added to the babel target.
+* **The `pbYield` helper was added and greedy rendering is disabled by default.**
+
+## API Changes
+
+* The `ADPOD` mediatype has received a deprecation warning.
+* Bid response helper methods (`getBidResponses*` and `getNoBids*`) now return an array which also exposes the array under `.bids` for backward compatibility.
+* `getHighestUnusedBidResponseForAdUnitCode` returns null instead of an empty object when no suitable bid exists.
+* **Schain data now lives under `ortb2.source.ext.schain` and is normalized when provided in first party data. The module is now superfluous for publishers transitioned to seeing this object directly and will be removed in the future.**
+* **Bid adapters supporting the `NATIVE` mediatype no longer send legacy targeting keys. Please migrate to the new setup.**
```

Comment:
```suggestion
* **Native ad units no longer support `sendTargetingKeys`, and native key-value pairs can no longer be sent to the ad server. Please migrate to a newer setup.**
```

`sendTargetingKeys` doesn't concern adapters - it's the second entry in [a table of choices](https://docs.prebid.org/prebid/native-implementation.html) (sigh) that publisher has on how to set up native creatives.